src/public/meeting.ts

```diff
@@ -325,4 +349,22 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel);
     sendMessageToParent('meeting.getAppContentStageSharingState', callback);
   }
+
+  /**
+   * Registers a handler for changes to speaking array.
+   * @param handler The handler to invoke when the array of participants speaking changes. A value
+   * of 0 (no participants speaking) or 1 (one or more participants are speaking) is provided.
+   * @param includeParticipants is a flag to opt-in to receive participant ids of who is speaking.
+   * Default value is false.
+   */
+  export function registerDetectSpeakingStateChangedHandler(
+    handler: (isSpeakingDetected: ISpeakingState) => void,
```

Comment:
The parameter name being `isSpeakingDetected` is confusing, because it's actually `ISpeakingState`, so you'd have to do `isSpeakingDetected.isSpeakingDetected` to actually get a `true`/`false` value. That doesn't seem right.

Reply:
Thanks Marcin! The idea for ISpeakingState was to communicate if anyone is speaking, and also (if the developer asks and has the necessary permission) to send an array of all the participant ids and their `isSpeaking` (0 or 1). Maybe a better name for this parameter would be `speakingState`? And if a developer only wants to know if _anyone_ is speaking they can do `speakingState.isSpeakingDetected`.

In the case of the developer wanting participant info too, do you think it would make more sense to return an array of just the ids of participants that are currently speaking or an array of all participants and whether or not they are speaking? Would be happy to discuss on a call too if you'd like.

Reply:
+1 for speakingState instead of isSpeakingDetected for the response object.

Reply:
@MarcinJuraszek @ryanbliss -- I updated this parameter name to `speakingState` to hopefully be clearer.